server/src/lsp-connection.ts

```diff
@@ -28,15 +35,18 @@ export function createLspConnection(options: IServerOptions): lsp.IConnection {
         lspClient,
         tsserverPath: options.tsserverPath,
         tsserverLogFile: options.tsserverLogFile,
-        tsserverLogVerbosity: options.tsserverLogVerbosity
+        tsserverLogVerbosity: options.tsserverLogVerbosity,
+        ignoredDiagnosticCodes: options.ignoredDiagnosticCodes,
     });
 
     connection.onInitialize(server.initialize.bind(server));
 
     connection.onDidOpenTextDocument(server.didOpenTextDocument.bind(server));
     connection.onDidSaveTextDocument(server.didSaveTextDocument.bind(server));
     connection.onDidCloseTextDocument(server.didCloseTextDocument.bind(server));
-    connection.onDidChangeTextDocument(server.didChangeTextDocument.bind(server));
+    connection.onDidChangeTextDocument(
+        server.didChangeTextDocument.bind(server)
+    );
```

Comment:
Was this formatting change necessary? (Also below this one)

Reply:
Indeed, a mistake I was wrestling with my Prettier plugin :P
Will amend.

Reply:
Did [amend](https://github.com/pyrho/typescript-language-server/blob/feat/ignore-diagnostics/server/src/lsp-connection.ts) this change, but this PR isn't updated yet, not sure if it's github lagging or if I'm doing something wrong 😅

Reply:
It looks good now. The change doesn't appear in the diff anymore.